### PR TITLE
Tackling some Balance

### PR DESCRIPTION
**Pull request by GreatBigFailure**
Balances some aspects of tackling.

tl;dr: Tackle bonuses aren't as all or nothing. 

- Allows smaller and more specific bonuses. Essentially all deciding numbers are 10 times higher, then divided by 10 to allow for things like a +5 bonus, equivalent to the previously unusable 0.5 bonus. 
-  Tackle offense and defense all start at a base power of 50, rather than 0. This dilutes any bonuses to make them less all or nothing factors.
- The unavoidable knockdown you get when tackling is increased by 50%, so is the damage you receive for hitting something dense
- Adds chat messages to the tackler and tackle-y. Also adds sound effects for both hitting a mob and a dense object
- Balance tweaks to a bunch of bonuses. Notably the offense bonus of being fat is halved (3 -> 15, effectively 1.5)

I'm sure this won't be the last tweak but it's good progress.

-->
:cl:
 * tweak: Tackles are now, probably, more balanced. Things that provide a bonus to tackles aren't just guaranteed success.

### CODE REVIEW

**Comment by Servalfang** [comment]
I think the base shouldn't be 50. Maybe 25? 20?

**Comment by SonixApache** [comment]
:s chasing a greyshirt but he's proving hard to catch
[sec] what's his AC?
:s above 18
[sec] jesus okay i'll get the RPG out

**Comment by Eneocho** [comment]
As I asked on the original PR, can shorts also buff tackling like luchador masks do? It'd complete the proper luchador getup and most wrestlers use only shorts (or shorts and a sleeveless shirt) when wrestling, so it'd make some sense in that regard as well.

**Comment by GreatBigFailure** [comment]
@SonixApache It does feel weird talking about spess like a crpg. It's just easier to use those terms I guess. Tackling is decided like a stat based RPG combat system but you get the stats from stupid things like being a vegan clown. Honestly I shouldn't have advertised the bonuses as much, one reason I wanted them was because of how fun it was finding the tiny secret bonuses you can get to unarmed/disarm chance and anything tied to get_strength(). 

> As I asked on the original PR, can shorts also buff tackling like luchador masks do? It'd complete the proper luchador getup and most wrestlers use only shorts (or shorts and a sleeveless shirt) when wrestling, so it'd make some sense in that regard as well.

I'm of two minds on that. My gut tells me you can just wear the shorts to complete the aesthetic and it doesn't need a bonus since the mask and other things you could do would raise it. On the other hand shorts and the football jersey are in the same slot so you'd be torn between fulfilling the aesthetics of the gimmick vs making it work mechanically.
Actually, I have an idea that will satisfy both lines of thinking. Just need to wait for this to be merged so I don't incur git's wrath.

**Comment by GreatBigFailure** [comment]
>I think the base shouldn't be 50. Maybe 25? 20?

Honestly I think I might need to make it 100ish down the line but for now this is good, until the re-adjusted bonuses are tested at least. Here, numbers for two human assistants:

Base 50 + 50 from human + 10 from strength (always gives 1 default, doesn't go lower) so we're at 110 for both. Let's say the tackler got a running start, now he's at 130, if he didn't he'd be at 100.  So defense is rolling 55 - 110 vs offense's 65 - 130. Now let's say the offense is wearing a wrestling mask (+5) and drank the [redacted] strong boy drink (+10) and the defense is fat (+25). 
Offense: 72 - 145
Defense: 67 - 135

So the numbers have evened out more but still have an effect on the rolls. For now I think 50 is good but if more bonuses get added it may need to increase. I had wanted to make tackles very unpredictable by giving just about everything a tiny tackle effect but decided against it for a few reasons. I'd still like a lot of things that effect it though, just not a ridiculous amount.

**Comment by Kurfursten** [comment]
I think the mask makes more sense to give the bonus than the pants. The pants are just aesthetic, but wearing the mask causes your speech to be ALL CAPS so you are actually crippling your subtlety even over the radio when you wear it. And people are slightly less trustful of face-covering-mask-wearers.
